README.md

```diff
@@ -1,6 +1,6 @@
 # Harpoon
 
-Harpoon is a staking dashboard for Tezos built using ConseilJS. Harpoon collects metrics about each baker on the Tezos network and allows users to search for and evaluate bakers based off of these performance stats. This data includes blocks baked, missed, and stolen, stake relative to the rest of the network, as well as a letter grade based on a normalized scoring system. Harpoon also allows for a user to view and audit the rewards won/payed out by bakers.
+Harpoon is a staking dashboard for the Tezos blockchain built using [ConseilJS](https://github.com/Cryptonomic/ConseilJS). Harpoon collects metrics about each baker on the Tezos network and allows users to search for and evaluate bakers based on these performance stats. The data includes blocks baked, missed, and stolen, stake relative to the rest of the network, as well as a letter grade based on a normalized scoring system. Harpoon also enables baker auditing by showing the rewards won and payed out. [Cryptonomic](https://twitter.com/cryptonomictech) has a public deployment of [Harpoon](https://harpoon.arronax.io/) under the [Arronax](https://harpoon.arronax.io/) umbrella of blockchain analytics products.
```

Comment:
> If the verb pay is used in a nautical sense, the correct form is payed.

Wow I didn't know we'd stick to our nautical theme this hard
Should be 'paid' instead